pkg/controller/workload/workload_event.go

```diff
@@ -364,6 +363,8 @@ func handleDataWithoutService(workload *workloadapi.Workload) error {
 func handleWorkloadData(workload *workloadapi.Workload) error {
 	var err error
 
+	go workloadDataToCache(workload)
```

Comment:
same here, do we really need to run it concurrently. Even here, the underlying map need to protect, so it only run in serial

Reply:
fixed